package.json

```diff
@@ -32,129 +38,119 @@
 			"current node"
 		]
 	},
-	"dependencies": {
```

Comment:
I merged existing `dependencies` and `devDependencies` as `dependencies`. Reasons:

- Is a workaround for https://github.com/yarnpkg/yarn/issues/7807
- There was no clear distinction between `dependencies` and `devDependencies` anyway. And is not super important because the root package is private and not meant to be imported by anybody else.
- Given that the purpose of the root package is to run tests and build packages, one could argue that packages previously listed under `devDependencies` are actually `dependencies` because they are required for the primary purpose of the package.
- This all requires further cleanup anyway (delete unused deps, bubble up dependencies to the packages that actually uses them...)

Reply:
I'm glad to see this. It would be nice to have a linter that prevents `devDependencies` from being added. Not necessary for this PR.